google-cloud-tools-plugin/ultimate/src/com/google/cloud/tools/intellij/appengine/server/instance/AppEngineServerModel.java

```diff
@@ -162,14 +162,20 @@ public void checkConfiguration() throws RuntimeConfigurationException {
           "App Engine facet not found in '" + artifact.getName() + "' artifact");
     }
 
+    if (CloudSdkService.getInstance().getSdkHomePath() == null) {
+      throw new RuntimeConfigurationError(
```

Comment:
Yeah, I just removed the facet and the projects runs great. Which makes me wonder: should we actually remove that warning? Or are we going to have any use for a facet in local runs, in the future?

If dev_appserver doesn't include the SDK libraries by default, we could probably add them to the facet? Need to check this, though.